src/tools/setup-virtualenv.sh

```diff
@@ -22,12 +22,12 @@ virtualenv --python python2.7 $DIR
 . $DIR/bin/activate
 # older versions of pip will not install wrap_console scripts
 # when using wheel packages
-pip --log $DIR/log.txt install --upgrade 'pip >= 6.1'
+pip --disable-pip-version-check --log $DIR/log.txt install --upgrade 'pip >= 6.1'
```

Comment:
Which pip version doesn't have it?  I didn't realize that there were versions without it, since all versions I have access to, even pretty old ones, have it.